### ISSUE TITLE
Links on 404 page are not readable

### ISSUE THREAD

**Issue by janhassel** [carbon-design-system/gatsby-theme-carbon#1069]
## Detailed description

The links on the 404 page are white text on a light gray background and therefore almost invisible.
I believe the former version of this page was dark, hence the light font.

## Steps to reproduce the issue

1. Visit https://www.carbondesignsystem.com/404/

## Additional information

![image](https://user-images.githubusercontent.com/28265588/100198287-ef379b80-2efb-11eb-96df-24d6d88fa796.png)
